lib/themes/theme_base.dart

```diff
@@ -27,7 +27,7 @@ import 'package:cake_wallet/themes/extensions/transaction_trade_theme.dart';
 import 'package:cake_wallet/themes/extensions/wallet_list_theme.dart';
 import 'package:flutter/material.dart';
 
-enum ThemeType { light, bright, dark }
+enum ThemeType { dark, light, bright }
```

Comment:
I believe changing the order here will cause some issues as there are multiple places that probably depend on the index of those enum values, so since there is no need for them to be changed, please revert them